apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerParameter.cs

```diff
@@ -59,6 +59,9 @@ public override DbType DbType
                     case DbType.Int64:
                         SpannerDbType = SpannerDbType.Int64;
                         break;
+                    case DbType.VarNumeric:
```

Comment:
Would it be worth it to add a method in SpannerDbType.FromDbType and move all the code in this set there? Just for the sake of having type conversions in one place?

Reply:
+1 to this.

Reply:
Done.